tests/cli/test_init.py

```diff
@@ -103,28 +311,19 @@ def test_cli_init_on_existing_project_with_no_uncommitted_dirs_answering_yes_to_
     assert_no_logging_messages_or_tracebacks(caplog, result)
 
 
-@pytest.mark.xfail(
-    reason="This command is not yet implemented for the modern API",
-    run=True,
-    strict=True,
-)
 @mock.patch("webbrowser.open", return_value=True, side_effect=None)
 def test_cli_init_on_complete_existing_project_all_uncommitted_dirs_exist(
     mock_webbrowser,
     caplog,
     monkeypatch,
-    tmp_path_factory,
+    tmp_path,
 ):
-    """
-    This test walks through the onboarding experience.
-
-    The user just checked an existing project out of source control and does
-    not yet have an uncommitted directory.
-    """
-    root_dir: str = str(tmp_path_factory.mktemp("hiya"))
+    root_dir_path = tmp_path / "hiya"
+    root_dir_path.mkdir()
+    root_dir = str(root_dir_path)
+    data_folder_path = os.path.join(root_dir, "data")
     os.makedirs(os.path.join(root_dir, "data"))
-    data_folder_path: str = os.path.join(root_dir, "data")
-    data_path: str = os.path.join(root_dir, "data", "Titanic.csv")
+    data_path: str = os.path.join(data_folder_path, "Titanic.csv")
     fixture_path: str = file_relative_path(
         __file__, os.path.join("..", "test_sets", "Titanic.csv")
     )
```

Comment:
Nitpick: we don't need data fixtures for all `init` tests. Let's remove them for simplicity.

Reply:
YES good call.